src/main/java/racingcar/model/Car.java

```diff
@@ -50,4 +50,9 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(position, name);
     }
+
+    @Override
+    public int compareTo(Car car) {
+        return car.getPosition() - this.getPosition();
+    }
```

Comment:
```suggestion
    @Override
    public int compareTo(Car car) {
        return car.position - position;
    }
```

`Integer#compare` 를 사용하는 것도 방법이겠네요.